src/main/kotlin/racingcar/car/Car.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.car
+
+class Car(val name: Name, val position: Position = Position()) {
+
+    companion object {
+        fun from(name: String): Car {
+            return Car(Name(name))
+        }
+    }
```

Comment:
```suggestion
    companion object {
        fun withName(name: String): Car {
            return Car(Name(name))
        }
    }
```

```kotlin
Car.withName("tre")
```

I personally prefer to give a meaningful name to the factory functions, whenever it's possible.
You don't have to change it though.
Just wanted to share my preference :)

Reply:
Thank you for sharing your opinion.
Totally understand you. But I want to share my thought as well.
From my perspective, I thought the parameter gives us enough information so I used to name function without any hint for parameter. For me, it feels like kind of a duplicate information.
What do you say?

Reply:
I think that's also a good idea!